file_catalog/__main__.py

```diff
@@ -12,6 +14,22 @@ def main():
     parser.add_argument('--debug', action='store_true', default=False, help='Debug flag')
     args = parser.parse_args()
     kwargs = {k:v for k,v in vars(args).items() if v}
+
+    # create config dict
+    config = Config(os.path.abspath(os.path.join(os.path.dirname(__file__), '..', 'server.cfg')))
```

Comment:
The path assumption here seems dangerous. Probably better to pass it in as an argument to the parser.